### ISSUE TITLE
Oil troubles

### ISSUE THREAD

**Issue by Cadams111** [Deadlock989/IndustrialRevolution#180]
So while blueprinting oil stuff I came across two things that, while not issues, could (probably) easily be resolved. First, the bitumen icon. I think everyone can agree that bitumen is basically trash. Fair enough, trash is something that the base game is lacking in, and it does arguably have a use, but it’s usefulness is not my concern here. The issue is that bitumen is trash that looks like something that is not trash, coal. Yes the icons are slightly different, but coal is used around oil and when I go to get some coal it sucks to think you’re getting some coal and instead get an inventory full of trash.

The second issue is about the petro generators. When cracking light oil to petroleum, you get natural gas, which can be used for furnaces, ethanol, simply vented, or burned in a petro generator. Unfortunately the generator is a little hard to control, it makes a ton of power for very little gas and actually makes more power than my entire refinery uses. When the petro generators are not at full capacity, I’m using a little pump valve to vent excess gas, but I have enough generators that when powering my whole factory they should take priority over the coal burning generators as “free” power that I’m getting as a byproduct. Can you make the petro generator have a priority toggle like the battery discharger? I do get that there aren’t that many priority levels and if you’ve already considered this and decided against it, no pressure. 

TL;DR bitumen looks too much like coal and petro generators should have a priority toggle like battery generators.
 
As always thanks for the awesome mod!

**Comment by Deadlock989** [comment]
Bitumen icons are a variant of the ore icons which are high on my list of things to improve if time ever permits.

Generators can be controlled with vanilla mechanics (power switches, accumulator signals).
